packages/docusaurus-plugin-lint-api/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@docusaurus/plugin-lint-api",
+  "version": "2.0.0-alpha.58",
+  "description": "A lint for API docs",
+  "main": "src/index.js",
+  "license": "MIT",
+  "devDependencies": {
+    "markdownlint": "^0.20.3"
+  },
+  "peerDependencies": {
+    "markdown-it": "^10.0.0"
```

Comment:
requiring the user to add the deps is likely to be annoying
both should rather be "dependencies" as it's code the end user will run